static/app/components/u2f/u2finterface.tsx

```diff
@@ -258,6 +274,21 @@ class U2fInterface extends React.Component<Props, State> {
     return this.state.deviceFailure !== 'BAD_APPID';
   }
 
+  get buttonMessage() {
```

Comment:
What's the sentiment on just inlining the getter and button render function inside the render method? Since it's used only in a single place I think we should just go for some good ol straight forward code - it would basically become just
```tsx
<a onClick={this.onTryAgain} className="btn btn-primary">
   {this.props.flowMode === 'enroll' ? t('Enroll with WebAuthn') : t('Sign in with WebAuthn')}
</a>
```